Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.11-slim
+
+RUN useradd containeruser
+
+WORKDIR /home/containeruser
+
+RUN apt-get update && \
+    apt-get install -y libpq-dev gcc
+
+COPY app app
+COPY time_tracker.py config.py docker-entrypoint.sh requirements.txt ./
+RUN pip install -r requirements.txt \
+    && chmod +x docker-entrypoint.sh \
+    && chown -R containeruser:containeruser ./
+
+# Set environment variables
+ENV DATABASE_URL="postgresql://mash:mash@db:5432/time_tracker" \
+    MAILGUN_API_KEY="" \
+    MAILGUN_API_URL="" \
+    PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    REDIS_URL="redis://cache:6379" \
+    SECRET_KEY=27a832aa91849382e326b5200f2c3d748a78bac4587e6ffd626dd87d73438f44 \
```

Comment:
Should this be set here?

Reply:
Will be overridden in production environment, but maybe it should only be in the compose envs? I suppose if anyone wanted to run the container on its own, without compose, it would need to be set.

Reply:
I'm just thinking of someone who boots up the image and forgets to change this - that would leave a security vulnerability.